### PR TITLE
[weight-decay] Enable for batch normalization

### PR DESCRIPTION
**Pull request by kparichay**
Enable weight decay for batch normalization.

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM
